Makefile

```diff
@@ -3,12 +3,13 @@ PKG_CONFIG = pkg-config
 BASE_CFLAGS=-std=c99 -Wall -Wextra -O2 `$(PKG_CONFIG) --cflags gio-2.0 gio-unix-2.0 glib-2.0 mpv`
 BASE_LDFLAGS=`$(PKG_CONFIG) --libs gio-2.0 gio-unix-2.0 glib-2.0`
 
+DESTDIR := $(HOME)/.config/mpv/scripts
+
 mpris.so: mpris.c
 	$(CC) mpris.c -o mpris.so $(BASE_CFLAGS) $(CFLAGS) $(BASE_LDFLAGS) $(LDFLAGS) -shared -fPIC
 
 install: mpris.so
-	mkdir -p $(HOME)/.config/mpv/scripts
-	cp mpris.so -t $(HOME)/.config/mpv/scripts
+	install -Dt $(DESTDIR) mpris.so
```

Comment:
It is a good practice to make `install` overridable just like `CC` or `PKG_CONFIG`.

Reply:
LOL thanks for coming here @jtojnar. Then how do I make a _target_ overrideable? and how would that look in our expression?

Reply:
Oh I think you meant `install` the command :)...